app/views/chargeback/_cb_rate_edit.html.haml

```diff
@@ -64,3 +65,6 @@
               /if the rate detail have a metric associated, display an options field with per_unit selected
               %td
                 = select_tag("per_unit_#{i}", options_for_select(measure.measures, r.per_unit), "data-miq_observe" => {:url => url}.to_json)
+          /Show the code of the currency selected by the user
+          %td{:id => "column_currency_#{i}"}
+            = r.detail_currency.code
```

Comment:
@tledesma maybe you can store detail_currency in @sb[:rate_details] hash here https://github.com/ManageIQ/manageiq/blob/master/app/controllers/chargeback_controller.rb#L200 as you are already looping thru rate_details at that point.